src/app.js

```diff
@@ -1,21 +1,29 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const fs = require('fs');
+const path = require('path');
+
+const [sourcePath, directPath] = process.argv.slice(2);
+
+const moveFile = (from, to) => {
+  const sourceFile = path.join(__dirname, from);
+  const destinationDir = path.join(__dirname, to);
+
+  if (!fs.existsSync(destinationDir)) {
+    throw new Error(`directory '${destinationDir}' does not exist`);
+  }
+
+  const isDir = fs.lstatSync(destinationDir).isDirectory();
+
+  const finalPath = isDir
+    ? path.join(destinationDir, path.basename(sourceFile))
+    : destinationDir;
+
+  console.log('Moving file from:', sourceFile);
+  console.log('Moving file to:', finalPath);
+
+  fs.renameSync(sourceFile, finalPath);
```

Comment:
Use the 'fs.promises.rename' method instead of 'fs.renameSync' to avoid blocking the event loop.